DcShare/Controller/ShareViewController.swift

```diff
@@ -77,27 +77,25 @@ class ShareViewController: SLComposeServiceViewController {
     }
 
     override func presentationAnimationDidFinish() {
-        if dbHelper.currentDatabaseLocation == dbHelper.sharedDbFile {
-            dcContext.logger = self.logger
-            dcContext.openDatabase(dbFile: dbHelper.sharedDbFile)
-            isAccountConfigured = dcContext.isConfigured()
-            if isAccountConfigured {
-                if #available(iOSApplicationExtension 13.0, *) {
-                   if let intent = self.extensionContext?.intent as? INSendMessageIntent, let chatId = Int(intent.conversationIdentifier ?? "") {
-                       selectedChatId = chatId
-                   }
-                }
+        dcAccounts.openDatabase()
+        dcContext.logger = self.logger
+        isAccountConfigured = dcContext.isConfigured()
+        if isAccountConfigured {
+            if #available(iOSApplicationExtension 13.0, *) {
+               if let intent = self.extensionContext?.intent as? INSendMessageIntent, let chatId = Int(intent.conversationIdentifier ?? "") {
+                   selectedChatId = chatId
+               }
+            }
 
-                if selectedChatId == nil {
-                    selectedChatId = dcContext.getChatIdByContactId(contactId: Int(DC_CONTACT_ID_SELF))
-                }
-                if let chatId = selectedChatId {
-                    selectedChat = dcContext.getChat(chatId: chatId)
-                }
-                DispatchQueue.global(qos: .userInitiated).async { [weak self] in
-                    guard let self = self else { return }
-                    self.shareAttachment = ShareAttachment(dcContext: self.dcContext, inputItems: self.extensionContext?.inputItems, delegate: self)
-                }
+            if selectedChatId == nil {
+                selectedChatId = dcContext.getChatIdByContactId(contactId: Int(DC_CONTACT_ID_SELF))
+            }
+            if let chatId = selectedChatId {
+                selectedChat = dcContext.getChat(chatId: chatId)
+            }
+            DispatchQueue.global(qos: .userInitiated).async { [weak self] in
+                guard let self = self else { return }
+                self.shareAttachment = ShareAttachment(dcContext: self.dcAccounts.getSelected(), inputItems: self.extensionContext?.inputItems, delegate: self)
```

Comment:
why use `self.dcAccounts.getSelected()` if `self.dcContext` is available as well.

unless there is a good reason, maybe better use either the one or the other throughout the file. getSelected() has some risk to be changed, so maybe better use `self.dcContext`.